deps/libressl.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
```

Comment:
what's this?

Reply:
There's an issue when building for iOS where it doesn't link against this because it defines and empty static lib file, this is a fix for that.

Reply:
you don't need I think, you can force linking. try `--force_load` or something please.

Reply:
`-Wl,-force_load lib.a`

Reply:
Let me try that

Reply:
It's a different issue and unique to XCode where it doesn't even generate the lib<name>.a file in this case

Reply:
what's this crypto library anyway? It's kinda a while hack prone to break :(

Reply:
It's because some boost libraries reference it (asio/beast). Also rust uses it through openssl-sys on other non-apple platforms

Reply:
We're using a fixed version btw. (because rust requires it), so I don't think having a fix like this hurts.
We can have it included in a later version of the library and then we can remove the patch

Reply:
👍👍